# request
float64[] angles
float64 duration
---
# response
bool data
